package.json

```diff
@@ -47,6 +47,7 @@
 		"format:changed:main": "pretty-quick --branch main",
 		"format:changed:next": "pretty-quick --branch next",
 		"format:repo": "npm run prettier:repo:fix",
+		"generate:upcoming": "flub generate upcoming --releaseGroup client -t minor",
```

Comment:
If this is not a part of some more-commonly run script, and can cause PR build/CI failures if not run, then probably make sure this is well-communicated to the team.

Wondering if there's a "check" version of this that could be added to a "checks" script we've talked about?

Reply:
Yeah, "checks" is the answer to this long-term. I'll let people know that manual re-gen may be needed if CI fails.